aws-bundle/LICENSE

```diff
@@ -203,7 +203,7 @@
 
 --------------------------------------------------------------------------------
 
-This binary artifact contains code from the following projects:
+This binary artifact contains:
```

Comment:
Nit: Unnecessary change.

Reply:
OK, let me revert this part to avoid noise 😄

Reply:
Done !